application/Profile/LocalProfile.py

```
@@ -138,49 +164,60 @@
         else:
             nickname = None
 
+        logger.info(f"Successfully retrieved session info for {session_id}")
+
         return host, username, this_private_key_path, None, nickname
 
-    def set_session_nickname(self, session_id, nickname):
+    def set_session_nickname(self, session_id, nickname) -> Tuple[bool, str]:
```

Ruff (0.8.2)

Missing type annotation for function argument `session_id`

(ANN001)